src/reconcile/rec_visibility.c

```diff
@@ -39,6 +39,27 @@ __rec_update_save(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, WT_
     return (0);
 }
 
+/*
+ * __rec_delete_hs_upd_save --
+ *     Save a WT_DELETE_HS_UPDATE list to delete the update from the history store later.
```

Comment:
`WT_DELETE_HS_UPDATE` -> `WT_DELETE_HS_UPD`

Reply:
Could we say:
"Save an update into a WT_DELETE_HS_UPDATE list"? I am not sure if it's necessary to say that it is an update that needs to be deleted from the HS later. This function only serves the purpose of adding some content to a `WT_DELETE_HS_UPD`.

Reply:
I would still keep the history store here to make the intention clear.